<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_principale</class>
 <widget class="QMainWindow" name="interface_principale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>437</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>489</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>interface_principale</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>791</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>437</width>
       <height>24</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>326</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>800</width>
       <height>800</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>800</width>
       <height>800</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>120</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>535</x>
       <y>150</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="button_parametre">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>326</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>parametre</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_information">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>300</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitter">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>350</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>quitter</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
